README.md

```diff
@@ -1,38 +1,53 @@
 # Azure SDK for .NET
+This repository contains official .NET libraries for Azure services. You can find NuGet packages for these libraries [here](packages.md). 
```

Comment:
I think I'd still keep the documentation link higher here and if you feel like you do not want to repeat it then delete it from the Need help? section instead - seems more intuitive to search for it in the project description than issue reporting section I think.

Reply:
I will be completely honest, I am hesitant to merge a PR with a link to docs that are not yet working. I would like this README to be always in "good state". But when I follow the docs link today, I takes the users to a page that does not yet work.

Reply:
we simply do not have documentation out for preview packages yet so that is why it is pointing to a generic API docs site with all of the documentation. Since the new packages are not yet announced people looking for the old packages should be fine and I will rewire the link as soon as we publish the new docs on Friday/Monday